### PR TITLE
chore: disable analytics, add privacy declaration, bump version

### PR DESCRIPTION
**Pull request by bryce-mcmath**
Found this StackOverflow post regarding the recent notices from the App Store: https://stackoverflow.com/a/78168190

This led me to making a Privacy manifest following the recommendations here: https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api

You can see which reasons were chosen for each notice we received in the `PrivacyInfo.xcprivacy` file and compare with the other reason options at the above link.

Also in this PR is some iOS and Android specific settings to disable Firebase Analytics

### CODE REVIEW

File: app/ios/PrivacyInfo.xcprivacy [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+			<array>
+				<string>0A2A.1</string>
+			</array>
+		</dict>
+		<dict>
+			<key>NSPrivacyAccessedAPIType</key>
+			<string>NSPrivacyAccessedAPICategoryDiskSpace</string>
+			<key>NSPrivacyAccessedAPITypeReasons</key>
+			<array>
+				<string>E174.1</string>
```
Comment (jleach) [inline app/ios/PrivacyInfo.xcprivacy:20]: Resonable.

File: app/ios/PrivacyInfo.xcprivacy [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+			</array>
+		</dict>
+		<dict>
+			<key>NSPrivacyAccessedAPIType</key>
+			<string>NSPrivacyAccessedAPICategorySystemBootTime</string>
+			<key>NSPrivacyAccessedAPITypeReasons</key>
+			<array>
+				<string>35F9.1</string>
```
Comment (jleach) [inline app/ios/PrivacyInfo.xcprivacy:28]: Resonable

File: app/ios/PrivacyInfo.xcprivacy [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+			</array>
+		</dict>
+		<dict>
+			<key>NSPrivacyAccessedAPIType</key>
+			<string>NSPrivacyAccessedAPICategoryUserDefaults</string>
+			<key>NSPrivacyAccessedAPITypeReasons</key>
+			<array>
+				<string>CA92.1</string>
```
Comment (jleach) [inline app/ios/PrivacyInfo.xcprivacy:36]: Good

File: app/ios/PrivacyInfo.xcprivacy [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+<plist version="1.0">
+<dict>
+	<key>NSPrivacyAccessedAPITypes</key>
+	<array>
+		<dict>
+			<key>NSPrivacyAccessedAPIType</key>
+			<string>NSPrivacyAccessedAPICategoryFileTimestamp</string>
+			<key>NSPrivacyAccessedAPITypeReasons</key>
+			<array>
+				<string>0A2A.1</string>
```
Comment (jleach) [inline app/ios/PrivacyInfo.xcprivacy]: I think "C617.1" might be better. I read "0A2A.1" to mean that BC Wallet is itself a 3rd-party SDK which we're not. This is probably used by AFJ to store the tails file and other caching. It notes "This reason may only be declared by third-party SDKs." Thoughts?

[Ref Doc](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api)

**Review by jleach — CHANGES_REQUESTED** [review]

File: app/ios/PrivacyInfo.xcprivacy (hunk 4 above)
Comment (cvarjao) [reply app/ios/PrivacyInfo.xcprivacy]: I agree with using `C617.1`

File: app/ios/PrivacyInfo.xcprivacy (hunk 1 above)
Comment (cvarjao) [reply app/ios/PrivacyInfo.xcprivacy:20]: LGTM

File: app/ios/PrivacyInfo.xcprivacy (hunk 2 above)
Comment (cvarjao) [reply app/ios/PrivacyInfo.xcprivacy:28]: I am okay with this reason. I don't think it is something we use directly though. Seems like a side effect of using react native, indy-vdr, askar

File: app/ios/PrivacyInfo.xcprivacy (hunk 3 above)
Comment (cvarjao) [reply app/ios/PrivacyInfo.xcprivacy:36]: LGTM

File: app/ios/PrivacyInfo.xcprivacy (hunk 4 above)
Comment (bryce-mcmath) [reply app/ios/PrivacyInfo.xcprivacy]: Agree and fixed. I misunderstood this one

**Review by jleach — APPROVED** [review]

**Pull request merged**